core/modules/filter/filter.module

```diff
@@ -2254,6 +2260,11 @@ function _filter_autop($text) {
  * @see filter_filter_info()
  */
 function _filter_autop_tips($filter, $format, $long = FALSE) {
+  // Don't show allowed HTML tags if the user can't enter HTML.
```

Comment:
This code comment should probably say "Don't show tips if CKEditor will automatically create paragraphs itself."